core/trino-main/src/test/java/io/trino/server/TestSessionPropertyDefaults.java

```diff
@@ -39,28 +44,34 @@
     @Test
     public void testApplyDefaultProperties()
     {
-        SessionPropertyDefaults sessionPropertyDefaults = new SessionPropertyDefaults(TEST_NODE_INFO);
+        SessionPropertyDefaults sessionPropertyDefaults = new SessionPropertyDefaults(TEST_NODE_INFO, new AllowAllAccessControlManager());
```

Comment:
Most chagnes here seem unrelated to `Inject AccessControl to SessionPropertyDefaults`

Reply:
Right, my bad. It should be in "Allow session properties override once in transaction"

Reply:
Fixed